main.py

```diff
@@ -1,10 +1,53 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    Player.objects.all().delete()
+    file_name = "players.json"
+    with open(file_name) as f:
+        players_dict = json.load(f)
+
+    for name, value_player in players_dict.items():
+        Race.objects.get_or_create(
+            name=value_player["race"]["name"],
+            description=value_player["race"]["description"]
+        )
+        race_ = Race.objects.get(
+            name=value_player["race"]["name"],
+            description=value_player["race"]["description"]
+        )
```

Comment:
`get_or_create` returns a created instance. So, you don't need the next code
`        race_ = Race.objects.get(
            name=value_player["race"]["name"],
            description=value_player["race"]["description"]
        )`
 Fix it here and in other place too